spec/fixtures/users.yml

```diff
@@ -2,6 +2,7 @@ satoshi:
   display_name: satoshi
   email: satoshi@allthebits.com
   password_digest: <%= BCrypt::Password.create('satoshi', cost: 5) %>
+  role: User
```

Comment:
How does this work? Seems like it should be "user". Uppercase User looks like a class name

Reply:
Yeah, I have a followup to fix this.